### PR TITLE
homework 3. Servlets

### PR DESCRIPTION
**Pull request by borysovdenys**

### CODE REVIEW

File: Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/DeleteQuiz.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import borysov.entity.*;
+
+@WebServlet("/DeleteQuiz")
+public class DeleteQuiz extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        ArrayList<Quiz> loadedQuizzes = (ArrayList<Quiz>) req.getSession().getAttribute("listOfQuizzes");
+        int id = Integer.valueOf(req.getParameter("id_for_delete_field"));
+        System.out.println(loadedQuizzes);
```
Comment (dinamio) [inline Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/DeleteQuiz.java]: All okay, but System.out.println is bad practice, better use loggers

File: Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/AddQuiz.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        String theme = request.getParameter("theme_field");
+        Quiz quiz = new Quiz(subject, theme);
+        listOfQuizzes.add(quiz);
+        request.getSession().setAttribute("listOfQuizzes", listOfQuizzes);
+        System.out.println("add" + listOfQuizzes);
+        response.sendRedirect("showQuizzes.jsp");
+    }
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
```
Comment (dinamio) [inline Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/AddQuiz.java]: why do you need this method?

**Review by dinamio — APPROVED** [review]
Looks good

File: Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/DeleteQuiz.java (hunk 1 above)
Comment (borysovdenys) [reply Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/DeleteQuiz.java]: Yeah, I know. I haven't  plugged it yet.

File: Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/AddQuiz.java (hunk 2 above)
Comment (borysovdenys) [reply Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/AddQuiz.java]: It was created automatically

**Comment by borysovdenys** [comment]
So, could I merge it?

File: Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/AddQuiz.java (hunk 2 above)
Comment (dinamio) [reply Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/AddQuiz.java]: So delete it:)

File: Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/DeleteQuiz.java (hunk 1 above)
Comment (dinamio) [reply Borysov/homework 3. Servlets/Quiz/src/main/java/borysov/controller/DeleteQuiz.java]: So plug it:)

**Comment by dinamio** [comment]
Yes, you can merge, but better to fix issues, that I mentioned

**Pull request merged**